protobuf/model_config.proto

```diff
@@ -1869,6 +1869,80 @@ message ModelResponseCache
   bool enable = 1;
 }
 
+//@@
+//@@  .. cpp:var:: message ModelMetrics
+//@@
+//@@     The metrics setting of this model. Consider reusing this message body
+//@@     for backend metric custom configuration.
+//@@
+message ModelMetrics
+{
+  //@@
+  //@@  .. cpp:var:: message MetricControl
+  //@@
+  //@@     Override metrics settings of this model.
+  //@@
+  message MetricControl
+  {
+    //@@
+    //@@  .. cpp:var:: message MetricIdentifier
+    //@@
+    //@@     Specify metrics to be overridden with metric_option.
+    //@@
+    message MetricIdentifier
+    {
+      //@@  .. cpp:var:: string family
+      //@@
+      //@@     The name of the metric family to override with the custom value.
+      //@@     Check the list of supported metric families
+      //@@
+      // https://github.com/triton-inference-server/server/blob/main/docs/user_guide/metrics.md#histograms
```

Comment:
Fix the formatting/spacing
```suggestion
      //@@ https://github.com/triton-inference-server/server/blob/main/docs/user_guide/metrics.md#histograms
```

Reply:
It won't pass the pre-commit.
<img width="1030" alt="Screenshot 2024-11-04 at 12 16 11 AM" src="https://github.com/user-attachments/assets/b75a6668-2738-40f8-b03d-09620c63c71b">

Reply:
It is not obvious to the reader what to look for after redirecting to the page.
> Check the list of supported metric families

After reading this, I expect to see a list of dedicated metrics that allow customization, but I only see a summary of histograms. You should state that "all histograms reported by Triton can be customized" and indicate that the link is redirecting to all Triton exposed metrics.

Reply:
The link points to main branch which has not merged the dev branch.
Please refer to https://github.com/triton-inference-server/server/blob/yinggeh-DLIS-7457-per-model-metric-config/docs/user_guide/metrics.md#histograms

Reply:
Can ignore my comment if pre-commit doesn't like it

Reply:
Updated comment based on the suggestion. @GuanLuo